GDJS/Runtime/layer.ts

```diff
@@ -268,15 +263,51 @@ namespace gdjs {
 
     /**
      * Convert a point from the canvas coordinates (for example,
-     * the mouse position) to the scene coordinates.
+     * the mouse position) to the container coordinates.
      *
      * @param x The x position, in canvas coordinates.
      * @param y The y position, in canvas coordinates.
      * @param cameraId The camera number. Currently ignored.
-     */
-    convertCoords(x: float, y: float, cameraId?: integer): FloatPoint {
-      x -= this._cachedGameResolutionWidth / 2;
-      y -= this._cachedGameResolutionHeight / 2;
+     * @param result The point instance that is used to return the result.
+     */
+    convertCoords(
+      x: float,
+      y: float,
+      cameraId: integer = 0,
+      result: FloatPoint
+    ): FloatPoint {
+      // The result parameter used to be optional.
+      let position = result || [0, 0];
+      // TODO EBO use an AffineTransformation to avoid chained calls.
+      position = this._runtimeScene.convertCoords(x, y, position);
+      return this.applyLayerInverseTransformation(
+        position[0],
+        position[1],
+        cameraId,
+        position
+      );
+    }
```

Comment:
It's probably because the implementation do 2 calls instead of the same thing but inlined.